arrow-docs/docs/quickstart/libraries/README.md

```diff
@@ -72,15 +72,16 @@ For all the new typeclasses, it also includes the extensions available for basic
 
 Dependency: `arrow-core`
 
-### arrow-fx
+### arrow-fx-coroutines
 
 ```groovy
 dependencies {
-    implementation "io.arrow-kt:arrow-fx:$arrow_version"
+    implementation "io.arrow-kt:arrow-fx-coroutines:$arrow_version"
 }
 ```
 
-The [fx library]({{ '/fx/' | relative_url }}) offers a powerful concurrency DSL with an emphasis on easy concurrency and parallelism with guarantees about concurrent and parallel resource safety. It can be used with Arrow Fx's IO or a set of typeclasses to abstract over concurrency frameworks like `RxJava`, `Reactor`, `Arrow's IO`, etc.
+The [fx library]({{ '/fx/' | relative_url }}) is a Kotlin idiomatic functional effects stytem that offers a powerful set concurrency combinators and primitives with an emphasis on easy concurrency and parallelism with guarantees about concurrent and parallel resource safety and composition.
```

Comment:
Concurrency is mentioned twice in the description. I'd go with operators instead of combinators too.